sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -599,6 +602,309 @@ public async Task HandlerPropertiesCanBeSetAfterEventProcessorHasStopped()
             Assert.That(() => processor.ProcessErrorAsync += eventArgs => Task.CompletedTask, Throws.Nothing);
         }
 
+        /// <summary>
```

Comment:
Just a small comment to thank you for these tests. I really liked your approach to testing the ownership claim/steal.

Reply:
Thanks!